rgw/admin/user.go

```diff
@@ -46,6 +48,12 @@ type UserKeySpec struct {
 	SecretKey string `json:"secret_key" url:"secret-key"`
 }
 
+type userStat struct {
+	Size       *uint64 `json:"size"`
+	SizeRound  *uint64 `json:"size_rounded"`
```

Comment:
Why is the name here `SizeRound` and not `SizeRounded` generally, the rgw/admin stuff has been doing a good job having the json field names and the go field names match. This seems like an unnecessary exception to that.